VContainer/Assets/VContainer/Runtime/VContainer.asmdef

```diff
@@ -15,12 +15,19 @@
     "versionDefines": [
         {
             "name": "com.unity.entities",
+            "expression": "",
             "define": "VCONTAINER_ECS_INTEGRATION"
         },
+        {
+            "name": "com.unity.entities",
```

Comment:
Alternatively we could check for com.unity.entities.graphics, as this is required in 1.0 and doesnt exist before

Reply:
Question:
- Is it possible to use only `com.unity.entities.graphics` without  `com.unity.entities`? 
    - If `com.unity.entities` is always added to the package, I don't see the point of adding B here.

Reply:
`com.unity.entities.graphics` depends on `com.unity.entities`. 

```json
    "com.unity.entities.graphics": {
      "version": "1.0.0-exp.14",
      "depth": 0,
      "source": "registry",
      "dependencies": {
        "com.unity.entities": "1.0.0-exp.12",
        "com.unity.render-pipelines.core": "14.0.3"
      },
      "url": "https://packages.unity.com"
    },
```